{{cookiecutter.repostory_name}}/nginx/monitoring_certs/README

```diff
@@ -0,0 +1,2 @@
+Obtain ssl cert and key for this deployment and place them here, named `cert.crt` and `cert.key`. Also get the ca cert
+used to verify the client fetching monitoring data and place it here, name `ca.crt`.
```

Comment:
Sorry, with this instruction I won't be able to deploy myself. I don't get where I get the certificates from :(